lib/axp209.js

```diff
@@ -85,11 +95,38 @@ AXP209.prototype._readButton = function() {
   return (value ? 1 : 0);
 };
 
+AXP209.prototype._readVcc18Adc = function() {
+  // enable 12-bit ADC for VCC 1.8 
+  var p = this._i2c.readRegister(0x85, 1)[0]; // 0-2v range or .7 - 2.7v range?
+  var rhigh = this._i2c.readRegister(0x64, 1)[0].toString(16);
+  var rlow = this._i2c.readRegister(0x65, 1)[0].toString(16);
+  var r = parseInt(rhigh + rlow, 16);
```

Comment:
I'm not too familiar with bitwise operators,  but I tried this out and it gives me a different max value for the sensor.   Instead of going to `~19995`,   it goes up   `318823.2421875`  with the above code.

The code I was using is based off the bash script in the thread I linked.   But maybe I interpreted it wrong and yours is right.

I actually don't think it matters what the output value is,  only that we can scale it properly  and figure out how to make the `threshold` option work properly with this particular setting